src/views/simulation/resultats.vue

```diff
@@ -122,23 +118,46 @@ export default {
     Recapitulatif,
     ErrorsEmailAndSmsModal,
   },
-  mixins: [ResultatsMixin, StatisticsMixin],
+  mixins: [StatisticsMixin],
+
   setup() {
+    const resultsStore = useResultsStore()
+    const benefits = computed(() => resultsStore.benefits)
```

Comment:
Same que pour `resultats-details` sur le `computed` dans l'option api

Reply:
Pour celui là j'ai tout mis en composition api ici : https://github.com/betagouv/aides-jeunes/pull/4132

Reply:
Je serais d'avis de ne pas le faire ici, ça pollue pas mal l'historique non ? cest deux PR semble très imbriqué déjà. Faudra p.e. les merger si les deux font de la refacto et que la seconde casse ce que fait la première, t'en pense quoi ? (je serais p.e. pas validateur mais jte laisse mon avis :-) )

Reply:
Oui, ça pollue trop et c'est bien pour ça que j'ai fait la PR secondaire, je mergerais probablement plus tard la seconde avec une itération nécessaire pour réadapter si y a de nouvelles modifs

Reply:
Ah donc tu veux merge les deux PR ensemble avant de les merge dans main ?

Reply:
L'une après l'autre